packages/pigeon/platform_tests/ios_unit_tests/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -7,31 +7,31 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		0D02163D27BC7B48009BD76F /* nullable_returns.gen.m in Sources */ = {isa = PBXBuildFile; fileRef = 0D02163C27BC7B48009BD76F /* nullable_returns.gen.m */; };
```

Comment:
These project changes are all mechanical renames due to switching over to camel-casing for ObjC generation.